hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -132,9 +132,11 @@ public void init() throws DBException {
     }
 
     try {
-      THREAD_COUNT.getAndIncrement();
       synchronized(THREAD_COUNT) {
-        connection = ConnectionFactory.createConnection(config);
+        THREAD_COUNT.getAndIncrement();
```

Comment:
Fixed it up so that:
``private static final AtomicInteger THREAD_COUNT = new AtomicInteger(0)``
is now
`` private static Integer threadCount = 0``

and locking on that since we have to have a non-null object to synchronize on. And check style forced the name change :(